src/cli/output/installDependencies.ts

```diff
@@ -5,6 +5,8 @@ import path from "path";
 import { COMMON_TEMPLATES_ROOT } from "../../constants.js";
 import getPackageManager from "../../utils/getPackageManager.js";
 import installPackages from "../../utils/installPackages.js";
+import * as dotenv from "dotenv"; 
```

Comment:
I think we don't need to include dotenv multiple times. It's enough to import it and configure one time in index.ts

Reply:
Usually yes, but it was not accessible when I was using the value in another file even with a call primary in index.ts.

I've switched all the variables in constants.ts and got them in the application via this file.
constants.ts is the only place where the dotenv module will be called in this case and got information from the .env.
I'm currently on some code play with the variable to get them more concise and easy to understand and reuse.
I've done too many modifications, so I will send you a message on Twitter when I will be done with all the pieces of stuff.